metrics-exporter-log/src/lib.rs

```diff
@@ -26,77 +25,70 @@ pub struct LogExporter<C, R> {
     controller: C,
     recorder: R,
     level: Level,
+    interval: Duration,
+}
+
+/// The arguments used to construct a `LogExporter` instance.
+pub struct LogExporterArgs<C, R> {
+    /// The handle used to aquire metrics.
+    pub controller: C,
+    /// The `Recorder` formats the collected metrics.
+    pub recorder: R,
+    /// The configurable logging level.
+    pub level: Level,
+    /// The interval at which to log.
+    pub interval: Duration,
 }
 
 impl<C, R> LogExporter<C, R>
 where
-    R: Recorder + Clone + Into<String>,
+    R: Recorder + Flush + Clone + Export<String>,
```

Comment:
Don't think Clone is needed here.
I also expect Flush to imply Recorder.

Reply:
Right. Also I'm thinking of renaming `Flush` for something like `FlushableRecorder` so that its obvious. Its a bit long though.